src/seraphis_main/enote_finding_context.h

```diff
@@ -84,4 +89,131 @@ class EnoteFindingContextLedger
         const std::uint64_t chunk_max_size) const = 0;
 };
 
+////
+/// LegacyUnscannedTransaction: a transaction that is ready to be legacy view scanned
+///
+struct LegacyUnscannedTransaction final
+{
+    rct::key transaction_id;
+    uint64_t unlock_time;
+    sp::TxExtra tx_memo;
+    uint64_t total_enotes_before_tx;
```

Comment:
This doesn't capture all the information needed to index legacy enotes. It might be better to make this `std::vector<legacy_output_index_t>`, one for each enote after #42 is merged.

Reply:
Ya, it'll need that PR. I have a TODO in `scan_context_async_mock` for it